uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-comments-tab.component.js

```diff
@@ -154,21 +152,19 @@ define([
     }
 
     /**
-     * Loads the contact id of the current logged in user.
+     * Loads the contact id of the currently logged in user.
      *
      * @return {Promise}
      */
     function loadLoggedInContactId () {
-      vm.loading.loggedInContactId = true;
+      vm.loading.component = true;
```

Comment:
Decided to leave it since this is a part of function and it should block UI until session is updated.